CHANGELOG.md

```diff
@@ -4,11 +4,13 @@
 
 ### Added
 - Added switch for canary deplyment #44
+- Added resource variables #51
 
 ### Changed
 - Removed box example [no issue]
 - Updated README with intentions section #46
 - Proper rendering of credentials #45
+- Updated box version #49
```

Comment:
I think I'd like to include this. I imagine somebody going back to look for when something was removed for example.
```suggestion
- Updated box version #49
- Removed `dev` target from makefile
```

Reply:
Fixing manually